.travis.yml

```diff
@@ -1,30 +1,23 @@
 language: java
 jdk:
 - openjdk11
-
```

Comment:
this looks like the main issue.  the rest of the edits are probably not needed.

Reply:
I went looking through the travis CI requests and I saw failure to parse the yaml. It was complaining about not detecting a `-` after the script for the deploy job.

I separated out the script for the deploy to its own file and it seems to be working as intended. However, the deploy job is still running even though it has the `branch = master` condition. It might be executing still because of my before_script or before_install that lives outside of the job.